docs/references/backend/types/clerk-backend-api-response.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: ClerkBackendApiResponse
+description: An interface that describes the response of a Clerk backend API request.
+---
+
+# `ClerkBackendApiResponse`
+
+An interface that describes the response of a Clerk backend API request.
+
+```ts
+type ClerkBackendApiResponse<T> =
+  | {
+      data: T;
+      errors: null;
+      totalCount?: number;
+    }
+  | {
+      data: null;
+      errors: Array<ClerkAPIError>;
+      totalCount?: never;
```

Comment:
Is this actually comeback with `undefined` or `null` or is the property just not sent? 

`never` which indicates the values that will never occur, which is not true in this case, it will occur on a valid/error free response.

Reply:
oh woops let me fix lOL